flake.nix

```diff
@@ -4,28 +4,37 @@
   inputs = {
     nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
     flake-utils.url = "github:numtide/flake-utils";
+    ixx = {
+      # match version with npm package
+      url = "github:NuschtOS/ixx/v0.0.4";
```

Comment:
We should somehow assert this or add an update script or so

Reply:
Can this be done without embedding the version in the flake itself? Pasing the url? In development or should be fine to not use the exact same version .

Reply:
We cannot add more fanciness to the URL as that part of nix code is really restricted.

We probably need to add some assertion to the nix code which builds the package or so. or another update script.